code/parachain/frame/cosmwasm/src/runtimes/abstraction.rs

```diff
@@ -109,7 +109,7 @@ impl Gas {
 	}
 
 	fn current_mut(&mut self) -> &mut u64 {
-		self.checkpoints.last_mut().unwrap()
+		self.checkpoints.last_mut().expect("account by external caller")
```

Comment:
This description doesn’t make sense to me.  This never fails because there’s no code path in which checkpoints can be empty when current_mut is called.

Reply:
yes, that what i tried to write. there is method. somebody calls it. we assume he knows what he is doing

Reply:
will send prorper handing in next pr. so now unwraps or expects. guide from parity is no unwraps, i am for it.

Reply:
they write expect most of time. btw do not recall saw unwrap